### PR TITLE
yoga: xena merge

### PR DESCRIPTION
**Pull request by markgoddard**
- Update HAProxy alerting rules
- Revert "Build Neutron from stackhpc fork"
- Add workflows to build and promote overcloud host images (#509)

### CODE REVIEW

**Comment by Alex-Welsh** [comment]
Do we want both jammy and focal?

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -133,10 +133,6 @@ kolla_sources:
     type: git
     location: https://github.com/stackhpc/magnum.git
     reference: stackhpc/{{ openstack_release }}
-  neutron-base:
-    type: git
-    location: https://github.com/stackhpc/neutron.git
-    reference: stackhpc/{{ openstack_release }}
   neutron-base-plugin-networking-generic-switch:
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml]: As @mnasiadka pointed out, we need this in Wallaby.

**Comment by markgoddard** [comment]
> Do we want both jammy and focal?

I suppose we might as well.

**Comment by markgoddard** [comment]
> > Do we want both jammy and focal?
> 
> I suppose we might as well.

Could be a separate PR though

**Comment by Alex-Welsh** [comment]
> > > Do we want both jammy and focal?
> > 
> > 
> > I suppose we might as well.
> 
> Could be a separate PR though

Might as well add it since we're also adding Rocky 9

**Review by markgoddard — COMMENTED** [review]
+2

**Pull request merged**